nottshcData/nottshcData_package.Rmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "nottshcData package"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Template Function Skeleton}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = FALSE,
+  echo = TRUE
+)
+```
+
+# Background
+
+The CDU Data Science Team are using and building a R package called {nottshcData} that uses get_ and tidy_ functions to get data from a SQL data warehouse in a consistent way. Each table used in the data warehouse has corresponding functions but there are also functions for calculations. More was written about it by Milan Wiedemannon the [team blog](https://cdu-data-science-team.github.io/team-blog/posts/2021-08-06-nottshcverse/).
+
+The following code snippets are related to the files needed to connect to SQL servers and the message file upon connection. These files must be incorporated into a package to work.
+
+# Creating new SQL connections
+
+Add details to files `connect_sql.R` 
+
+```{r connect-sql}
+#' Connect to SQL servers
+#'
+#' @param driver String, specifying driver
+#' @param server String, specifying server, get overwritten at the moment,
+#' might need to revisit this functionality at some point later
```

Comment:
I don't actually understand what this means? doesn't seem dodgey but huh?

Reply:
I think they were notes but the function works ok for us so I'll just remove the notes.